### PR TITLE
Update reva to 1.5.1

### PR DESCRIPTION
**Pull request by IljaN**
Related issues:
https://github.com/owncloud/product/issues/139
https://github.com/owncloud/product/issues/254 (Partially)
https://github.com/owncloud/product/issues/279
https://github.com/owncloud/product/issues/179

### CODE REVIEW

File: .drone.env [hunk 1]
```diff
@@ -3,5 +3,5 @@ CORE_COMMITID=4a671a34a2f039c83520b5a72dd886b3a974a85e
 CORE_BRANCH=master
 
 # The test runner source for UI tests
-WEB_COMMITID=5878eeb7a9d7a1ef9259c1942d0ac8be57ecef02
-WEB_BRANCH=master
+WEB_COMMITID=7a2e2351b520e79a6203f083a7c7d257bdb3e7db
+WEB_BRANCH=update-reva-1.5.1
```
Comment (phil-davis) [inline .drone.env:7]: I don't understand why this is being done. Just set WEB_COMMITID to the latest commit in the master branch, and adjust the expected-failures here in OCIS repo t remove whatever scenarios now pass.

After this OCIS PR is merged, there will then be a new OCIS_COMMITID at the head of `owncloud/ocis` `master`. A PR can be done in `web` to use that commit id.

Comment (IljaN) [reply .drone.env:7]: This PR which updates reva removes expected failures which reside in ocis/web but are not yet merged to master.

Comment (IljaN) [reply .drone.env:7]: See https://github.com/owncloud/web/pull/4649

File: .drone.env [hunk 2]
```diff
@@ -3,5 +3,5 @@ CORE_COMMITID=4a671a34a2f039c83520b5a72dd886b3a974a85e
 CORE_BRANCH=master
 
 # The test runner source for UI tests
-WEB_COMMITID=5878eeb7a9d7a1ef9259c1942d0ac8be57ecef02
+WEB_COMMITID=7a2e2351b520e79a6203f083a7c7d257bdb3e7db
```
Comment (phil-davis) [inline .drone.env:6]: This is not a commit id in the master branch.

**Review by individual-it — APPROVED** [review]

File: .drone.env (hunk 1 above)
Comment (phil-davis) [reply .drone.env:7]: @IljaN if you change back to master branch of web and the latest web commit id, and rebase this PR to latest ocis master then this should pass. See #1382 where I tried that.
```
# The test runner source for UI tests
WEB_COMMITID=5945833ca594c16d33514da74b3d2eb64d3560fd
WEB_BRANCH=master
```

In PR https://github.com/owncloud/web/pull/4652 I have already removed the `webUITrashbinDelete/trashbinDelete.feature` bug-demo scenario that is no longer relevant.

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
I updated `WEB_COMMITID` to the latest. It was off-by-one. That will fix https://drone.owncloud.com/owncloud/ocis/2849/65/7
```
runsh: Total unexpected failed scenarios throughout the test run:
webUITrashbinDelete/trashbinDelete.feature:121
```

**Comment by phil-davis** [comment]
:tada:

**Comment by IljaN** [comment]
@phil-davis @individual-it Thanks you!

**Pull request merged**